linker/Mono.Linker.Steps/MarkStep.cs

```diff
@@ -1444,7 +1444,9 @@ bool ShouldParseMethodBody (MethodDefinition method)
 				return true;
 			case MethodAction.Parse:
 				AssemblyDefinition assembly = ResolveAssembly (method.DeclaringType.Scope);
-				return Annotations.GetAction (assembly) == AssemblyAction.Link;
+				return  (Annotations.GetAction (assembly) == AssemblyAction.Link) ||
+						(Annotations.GetAction(assembly) == AssemblyAction.Copy) ||
```

Comment:
I believe not checking for AssemblyAction.Copy here was a bug. @marek-safar, please confirm.

Reply:
Hmm, this really looks like a bug. Could you also use a switch here like `switch Annotations.GetAction (assembly) {`